packages/paste-core/components/sidebar/src/Sidebar.tsx

```diff
@@ -92,13 +107,23 @@ export const Sidebar = React.forwardRef<HTMLDivElement, SidebarProps>(
 
     return (
       <SidebarContext.Provider value={{collapsed, variant, sidebarId}}>
+        <SidebarSkipLinks
+          element={`${element}_SKIPLINKS`}
+          {...{
```

Comment:
suggestion(not-blocking): prefer
```
mainContentSkipLinkID={mainContentSkipLinkID}
sidebarNavigationSkipLinkID={sidebarNavigationSkipLinkID}
...
```
Easier to grok, doesn't need to make a temp new object in mem each re-render